src/electronApp/angularApp/components/status/status.component.html

```diff
@@ -48,18 +53,44 @@
                 <ng-template #cameraAvailableBlock>
                     <a class="accentColor" [routerLink]="'/camera'">View feed</a>
                 </ng-template>
-        
+
                 <ng-template #cameraNotAvailableBlock>
                     Not Available (<a class="accentColor" target="_blank" href='https://github.com/andycb/AdventurerClientJS/wiki/Using-The-Printer-Camera'>Help</a>)
                 </ng-template>
             </div>
         </td>
     </tr>
 
+    <tr>
+        <th>Control</th>
```

Comment:
I've been thinking about the UI in general and how we fit new features like this in. So I wonder if there's a way to add this without adding too much length. I'm thinking out loud here, so I'm open to your thoughts:

- Do we need all of the data that's already shown on the UI from older versions? Perhaps we should thinking about removing the Endstop field?
- Does the MoveMode need to be its own row, perhaps we could put it as an extension of the status like: "PRINTING_FROM_SD (Paused)"
- Do the pause / cancel buttons need their own row? Maybe they can span across the bottom next to the disconnect button without a header?

Super quick sketch I did: 
![image](https://user-images.githubusercontent.com/4417762/107120274-9cc41500-6884-11eb-94e2-0f84d80290b7.png)

Reply:
I first thought about adding the control buttons to the same row as the disconnect but I feel like the disconnect should stay in a corner away from the printer controls as it does not fit in with them (The control buttons are an interaction between the user and the printer and disconnect basically closes this interaction).
They in my mind are just two different things.

I've been thinking about UI a bit as well and I would remove the not changing stuff (like Firmware Version or Serial number) from the main page and just have the changing real-time information there (Temp and Status).

### I've made a quick design to show what I mean:
![Main Page design](https://user-images.githubusercontent.com/40121865/107134092-2c061280-68ef-11eb-9526-53ad026b5bd3.png)

I widened the application quite a bit as I wanted to add the camera feed to the main page as suggested [here](https://github.com/andycb/AdventurerClientJS/issues/28#issuecomment-761034655). This would probably make the other pages of the App look a bit empty.

In this design, there is a bit much room for the controls, but maybe there are still more to be added like for the light.

### With controls grayed out:
![Main Page with Controls grayed out](https://user-images.githubusercontent.com/40121865/107134251-9d929080-68f0-11eb-9a23-fc24a3d47cea.png)

### Switching to the static information
Could be done by clicking on the name or a link next to it (INFO) as this information is somewhat tied to the machine and is not needed to be checked on that often.
![Static information](https://user-images.githubusercontent.com/40121865/107134280-e1859580-68f0-11eb-8a01-b25f351a0f1f.png)

### With camera enabled
![Main Page with camera](https://user-images.githubusercontent.com/40121865/107134341-91f39980-68f1-11eb-9773-1dfaeffbfdb0.png)

Reply:
Those mock ups are awesome!  

I did come across some M commands to enable and disable the led.  I ran then through debug and they worked as expected.  If you want me to grab them and pay here, I can.

Reply:
Are they from [this site](https://take4-blue.com/en/3d-printer/adventurer3-control-3/)? Just found this as I was looking if there actually was a command for the led.
If you found something else it would surely be appreciated.

And a question, does the command to turn on/off the led work whilest a print is running?

Reply:
Yes that information is correct.
I noticed that I did not need to enter any RGB values at all to turn it off. Just the f0
To turn it on however I did need to set the rg&b's to 255.  I got curious to see if I could do different color combinations but it did not work. Only white
I was able to successfully do this during a print if I recall correctly.

Reply:
This is great. I really like this design, we should get it into the app.

One thought: The FlashForge Adventurer 3 Lite doesn't have a camera, it may be useful to have a way to hide the camera for them?

You're right about keeping the control buttons away from disconnect button, so shall we take your original change then pick up the work to implement this design in the next (v1.3) version? That way we can get the camera fixes released and take the time to make sure the other pages work with the wider design.

Reply:
With a987122fbd20687c2c60d4a09e6ef2a82a9ecf5e the buttons now get disabled if the printer isn't printing.

I've implemented all the changes you suggested in the comments except for [the MoveMode.ts](https://github.com/andycb/AdventurerClientJS/pull/39#discussion_r571438744) so I think this change should be ready for v1.2.
I agree that the UI change should be something for v1.3 as there are still a bunch of things that have to be worked out. Such as the fact that the _Adventurer 3 Lite_ doesn't have a camera. If the camera is hidden the UI would have to be changed along with that. The width of the App would probably not need to be as wide but then the layout has to be changed as well as with the design I posted before just cutting off the camera section would result in this:
![Status page without camera](https://user-images.githubusercontent.com/40121865/107161220-8492e980-699b-11eb-90ed-da09b3ea1e96.png)

Or maybe that space could be used by some other information
![Status page with live and static information](https://user-images.githubusercontent.com/40121865/107161295-0edb4d80-699c-11eb-90bb-1f3c3514e27c.png)


I think we should move the conversation about the UI change to a new issue or to the Discussions. Don't really know where best to put it.

Reply:
Height: Yeah, update it please. Its sometimes a bit of a fiddler to get it to work on both Windows and Mac for some reason, but I can tweak it as part of the release process it ifs an issue.

I think a discussion is the write place to continue to the UI conversation.